src/python/zquantum/core/interfaces/backend.py

```diff
@@ -108,6 +109,28 @@ def get_bitstring_distribution(
         return measurements.get_distribution()
 
 
+def split_circuit(
```

Comment:
Actually doesn't it belong more to the `circuits` module rather than `backend` ?

Reply:
3b7cc23 moves `split_circuits` into `zquantum.core.circuits`. Corresponding tests are also moved accordingly.